enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1220 @@
+          recurrence: <rrule|null>
+          # Given the identification of a date by an RRULE, at what time (always UTC) can the 
+          # permissive window begin. "00:00" if unset.
+          startTime: <time-of-day|null>
+          # Given the identification of a date by an RRULE, after what offset from the startTime should
+          # the permissive window close. This can create permissive windows within days that are not
+          # identified in the RRULE. For example, recurrence="FREQ=Weekly;BYDAY=Sa;",
+          # startTime="20:00", duration="8h" would permit material change initiation starting
+          # each Saturday at 8pm and continuing through Sunday 4am (all times are UTC). The default
+          # duration is 24:00-startTime (i.e. to the end of the day).
+          duration: <duration|null>
+          
+        
+        # Excluded date ranges override RRULE selections.
+        exclude:
+        # Dates should be specified in YYYY-MM-DD. Each date is excluded from 00:00 UTC for 24 hours.
+        - fromDate: <date>
+          # Non-inclusive until. If null, until defaults to the day after from (meaning a single day exclusion).
+          untilDate: <date|null>
+          # Provide additional detail for status when the cluster is within an exclusion period.
+          reason: Optional human readable which will be included in status description.
+
+```
+
+Permitted times (i.e. times at which the strategy enforcement state can be permissive) are specified using a 
+subset of the [RRULE RFC5545](https://www.rfc-editor.org/rfc/rfc5545#section-3.3.10) and, optionally, a
+starting and ending time of day. https://freetools.textmagic.com/rrule-generator is a helpful tool to
+review the basic semantics RRULE is capable of expressing. https://exontrol.com/exicalendar.jsp?config=/js#calendar
+offers more complex expressions.
+
+**RRULE Interpretation**
+RRULE supports expressions that suggest recurrence without implying an exact date. For example:
+- `RRULE:FREQ=YEARLY` - An event that occurs once a year on a specific date. 
+- `RRULE:FREQ=WEEKLY;INTERVAL=2` - An event that occurs every two weeks.
+
+All such expressions shall be evaluated with a starting date of Jan 1st, 1970 00:00Z. In other
+words, `RRULE:FREQ=YEARLY` would be considered permissive, for one day, at the start of each new year.
+
+If no `startTime` or `duration` is specified, any day selected by the RRULE will suggest a
+permissive 24h window unless a date is in the `exclude` ranges.
+
+**RRULE Constraints**
+A valid RRULE for change management:
+- must identify a date, so, although RRULE supports `FREQ=HOURLY`, it will be rejected if an attempt it made to use it.
+- cannot specify an end for the pattern. `RRULE:FREQ=DAILY;COUNT=3` suggests
+  an event that occurs every day for three days only. As such, neither `COUNT` nor `UNTIL` is 
+  supported and will be rejected if an attempt is made to use them.
+- cannot specify a permissive window more than 2 years away.

```

Comment:
RRULEs can definitely express "only on Saturdays & Sundays" or "only on the last Saturday of each month", etc. https://exontrol.com/exicalendar.jsp?config=/js#calendar helps build an intuition of exactly how expressive it can be. I think it would be difficult to achieve with more structured data. 
Cron is discussed in alternatives, so I will address in your comment there.